packages/dapp-high-roller/src/components/app-provider/app-provider.tsx

```diff
@@ -120,13 +120,9 @@ export class AppProvider {
     );
 
     if (state.stage === HighRollerStage.REVEALING) {
-      // TODO randomize this and save it in proposingPlayer state
-      const numberSalt =
-        "0xdfdaa4d168f0be935a1e1d12b555995bc5ea67bd33fce1bc5be0a1e0a381fc90";
-
       return await this.appInstance.takeAction({
         actionType: ActionType.REVEAL,
-        actionHash: numberSalt,
+        actionHash: state.salt,
```

Comment:
Seems like this should be the salt that was generated previously and held client-side which is now being used in the `REVEAL` stage, so I think we'd instead want:
```
actionHash: this.highRollerState.salt
```

Reply:
Added this in [latest commit](https://github.com/counterfactual/monorepo/pull/1356/commits/aa9bd2590152ebc884697e9cbe11e08f72a3dbee)